.rubocop.yml

```diff
@@ -25,6 +25,3 @@ Style/RegexpLiteral:
 BlockLength:
   Exclude:
     - './spec/*_spec.rb'
-
-Gemspec/RequiredRubyVersion:
```

Comment:
We need to re-add this for now (Whilst we have legacy ruby versions supported)

Reply:
ah ok